torch/utils/_config_module.py

```diff
@@ -156,6 +156,19 @@ def save_config(self) -> bytes:
             config.pop(key)
         return pickle.dumps(config, protocol=2)
 
+    def save_config_for_cache(self) -> bytes:
```

Comment:
It occurs to me that we could stick the whole dict into a field in the FxGraphDetails object rather than pickling it here and storing the serialized bytes in the details. The only advantage to doing that is the debug output we were inspecting today would show each field separately instead of just calling it "bytes"